modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
+
+.Description
+
+No detection of a virt-operator pod in the `Ready` state occurs in the past 10 minutes. The virt-operator deployment has a default replica of two pods.
+
+.Reason
+
+The virt-operator is the first Kubernetes Operator active in an OpenShift cluster. Its primary responsibilities are:
+
+* Installation
+* Live-update
+* Live-upgrade of a cluster
+* Monitoring the lifecycle of top-level controllers such as virt-controller, virt-handler, and virt-launcher
+* Managing the reconciliation of top-level controllers
+
+In addition, the virt-operator is responsible for cluster-wide tasks such as certificate rotation and some infrastructure management.
+
+[NOTE]
+====
+Virt-operator is not directly responsible for virtual machines in the cluster. Virt-operator's unavailability does not affect the custom workloads.
+====
+
+This alert indicates a failure at the cluster level. Critical cluster-wide management functionalities such as certification rotation, upgrade, and reconciliation of controllers are temporarily unavailable.
+
+.Troubleshoot
+
+. Check the status of the virt-operator deployment to learn more information. The following commands provide the associated events and show if there are any specific issues:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+
+oc -n $NAMESPACE get deploy virt-operator -o yaml
+
+oc -n $NAMESPACE describe deploy virt-operator
+----
+
+. Check if there are issues with the nodes for the control plane and masters, such as if they are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+.Resolution
+
+There are several reasons for no virt-operator pods being in a `Ready` state. Identify the root cause and take appropriate action.
+
+Otherwise, open a support issue and provide the information gathered in the troubleshooting process.
+
+//VirtAPIDown Alert
+[id="VirtAPIDown_{context}"]
+== VirtAPIDown alert
+
+.Description
+
+All OpenShift API servers are down.
+
+.Reason
+
+If all OpenShift API servers are down, then no API calls for OpenShift entities occur.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Verify that there are any running virt-api pods.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get pods -l kubevirt.io=virt-api
+----
+
+.Resolution
+
+Virt-api pods can be down for several reasons. Identify the root cause and take appropriate action.
+
+. Check the status of the virt-api deployment. Use these commands to learn about related events and show if there are any issues with pulling an image, a crashing pod, or other similar problems.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-api -o yaml
+
+oc -n $NAMESPACE describe deploy virt-api
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
```

Comment:
It would be good to include here, what all information needs to be collected for support case, if it comes to that, just like other alert's resolution section.